bcgov/moh-prime
# PRIME-1682 Health Authority Organization Registration: Care Type/Vendor Deletions

[PRIME-1682](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-1682)

### How to test:
1. Login as admin
2. Add the information needed for one of the health authorities (make sure you add more than one care type and vendor)
3. Create a health authority site (take note of the site's vendor and care type) (This may require that you create an authorized user and approve them before you can create/submit the site)
4. Submit site
5. Log in as admin again, go to **Site Registrations** >> **Health Authorities**
6. Click the kebab menu of the health authority you used in step one and click "Add Organization Information"
7. Edit **Health Authority Care Types:** 
-- Try to remove the care type that the site from step 4 is using (you should get a toast that says "Health authority care type could not be deleted")
-- Try to delete a care type that isn't used by any site (it should be deleted)
-- Click **Save and Continue** the updated care types list should be displayed in the **Organization Information**
-- Existing care types should not be editable
8. Edit the **Health Authority Vendors:**
-- Try to remove the care type that the site from step 4 is using (you should get a toast that says "Health authority vendor could not be deleted")
-- Try to delete a care type that isn't used by any site (it should be deleted)
-- Click **Save and Continue** the updated vendors list should be displayed in the **Organization Information**
-- Existing vendors should not be editable
